models/project/project.go

```diff
@@ -161,20 +161,20 @@ func (u *ProjectStruct) Initialize() {
 	u.database = interfaceDB
 }
 
-func (u *ProjectStruct) Save() bool {
+func (u *ProjectStruct) Save() error {
```

Comment:
__[golint]__
* [golint] exported method ProjectStruct.Save should have comment or be unexported